src/metabase/pulse/render/body.clj

```diff
@@ -834,13 +834,24 @@
       [:img {:style (style/style {:display :block :width :100%})
              :src   (:image-src image-bundle)}]]}))
 
+(defn- apply-funnel-viz-settings
+  [rows {order-data :funnel.rows}]
+  (let [rows (vec rows)]
```

Comment:
Huh. I never knew we could use the same variable name. In this case, the local `rows` is a vector of rows instead of a list (I presume?).

I think I've tried to declare a symbol inside `let` (or `for` I don't really remember) with the same name as another symbol outside `let`, but it never compiles. Maybe I've done something wrong 🤔